### ISSUE TITLE
How to exec on pod

### ISSUE THREAD

**Issue by BlueprintCEO** [iximiuz/client-go-examples#16]
It would be a nice example how to exec on a existing pod and run a curl from within it

Something like the following command :

kubectl exec -n nice_namespace nice_pod-8447f7cc77-jxz7d  -it -- /bin/bash

**Comment by iximiuz** [comment]
Yeah, it's been on my list for a while :) I should definitely come up with a minimal possible example of `kubect exec`-like functionality. Meanwhile, you can check this piece of code - it's doing pretty much that https://github.com/iximiuz/cdebug/blob/main/cmd/exec/exec_kubernetes.go.
